ingestion/tests/cli_e2e/test_cli_hive.py

```diff
@@ -153,7 +153,7 @@ def expected_filtered_schema_excludes() -> int:
 
     @staticmethod
     def expected_filtered_table_includes() -> int:
-        return 5
+        return 1
```

Comment:
Do we know why the count changed here?

Reply:
as it's a new instance, we have anything else on it, just a table, and a view